packages/io/stl-serializer/tests/progress.test.js

```diff
@@ -0,0 +1,17 @@
+const test = require('ava')
```

Comment:
nice ! 
Very good addition
although I believe 3 files for 3 tests is overkill , mostly as they are small tests